internal/services/recoveryservices/site_recovery_replicated_vm_resource.go

```diff
@@ -1059,6 +1127,84 @@ func waitForReplicationToBeHealthyRefreshFunc(d *pluginsdk.ResourceData, meta in
 	}
 }
 
+func expandSiteRecoveryReplicatedVMIPConfig(nicInput map[string]interface{}) []replicationprotecteditems.IPConfigInputDetails {
+	output := []replicationprotecteditems.IPConfigInputDetails{}
```

Comment:
minor, but we generally use `make` to initialize new slices

```suggestion
	output := make([]replicationprotecteditems.IPConfigInputDetails, 0)
```